shared/common-adapters/avatar.shared.js

```diff
@@ -0,0 +1,11 @@
+// @flow
+import type {Props} from './avatar'
+
+export function createUrl (props: Props): ?string {
```

Comment:
(I think flow is powerful enough to determine that subtype relation, if not don't worry this is good too)